spec/inputs/lumberjack_spec.rb

```diff
@@ -4,54 +4,46 @@
 require 'logstash/inputs/lumberjack'
 require "logstash/codecs/plain"
 require "logstash/codecs/multiline"
+require "logstash/event"
+require "lumberjack/client"
 
 describe LogStash::Inputs::Lumberjack do
-
-  let(:ssl_cert) { Stud::Temporary.pathname("ssl_certificate") }
-  let(:ssl_key)  { Stud::Temporary.pathname("ssl_key") }
-
+  let(:connection) { double("connection") }
+  let(:certificate) { LogStashTest.certificate }
+  let(:port) { LogStashTest.random_port }
```

Comment:
Longer term, we can use Flores for this, Flores::Random.tcp_listener will get you a socket listening on a random port, and we can use rspec mocks to inject this tcp socket into the Lumberjack Server. :)